core/src/test/java/org/elasticsearch/common/settings/loader/duplicate-settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
do we need this in a separate file? it could be inline in the test as a string stream?